ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.15.yaml

```diff
@@ -120,13 +120,19 @@ tests:
   steps:
     cluster_profile: openstack-hwoffload
     workflow: openshift-e2e-openstack-hwoffload-cgroupsv1
-- as: e2e-openstack-upi
+- as: e2e-openstack-dualstack-upi
   minimum_interval: 240h
   steps:
-    cluster_profile: openstack-vexxhost
+    cluster_profile: openstack-hwoffload
     env:
+      CONFIG_TYPE: dual-stack-upi
       OPENSTACK_TEST_SKIPS: The OpenStack platform creates Worker nodes in a server
         group\|The OpenStack platform creates Control plane nodes in a server group
+      OS_SUBNET_V6_RANGE: 2001:db8:2222:5555::/64
+      TEST_SKIPS: Services Secondary IP Family \[LinuxOnly\] should function for service
```

Comment:
This is an antipattern, we shouldn't skip tests in the job config. If this tests are not meant to pass in a dual stack environment, we should skip them dynamically in origin.

Reply:
@mandre this are not passing only in UPI.
With IPI these tests scenarios are passing, so I don't think we can skip them on origin.
I already opened two bugs to investigate that.
https://issues.redhat.com/browse/OCPBUGS-30047
https://issues.redhat.com/browse/OCPBUGS-30154

Reply:
Results from periodic IPI https://prow.ci.openshift.org/job-history/origin-ci-test/logs/periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.15-e2e-openstack-dualstack